src/ui_utils.c

```diff
@@ -95,7 +95,8 @@ int lookup_key(char *capability) {
 
 /** lookup all needed special keys not defined by ncurses */
 void lookup_keys() {
-    int not_ok = 0;
+    int alt_not_ok = 0;
```

Comment:
Pse use bool type.

Reply:
It's been like that before, but it's bool now. Thanks!

Reply:
(we are trying gradually improve code quality as we touch it. One of common topics is to use bool type if it's really meant to be a bool, see e.g. #290. This also involves using true/false literals and converting checks like ` if (a == 1)` into plain `if (a)`.)